src/SearchParameters/index.js

```diff
@@ -1149,13 +1149,24 @@ SearchParameters.prototype = {
    * @param  {string} facet the facet to refine
    * @param  {string} value the associated value
    * @return {SearchParameters}
-   * @throws will throw an error if the facet is not declared in the settings of the helper
+   * @deprecated since version 2.19.0, see {@link AlgoliaSearchHelper#toggleConjunctiveFacetRefinement}
```

Comment:
I think this link is wrong, should this be SearchParameters#toggleConjunctive... ?